docs/schema.md

```diff
@@ -636,6 +647,7 @@ The defined ecosystems are:
 | `Bioconductor` | The biological R package ecosystem. The `name` is an R package name. |
 | `Bitnami` | Bitnami package ecosystem; the `name` is the name of the affected component. |
 | `ConanCenter` | The ConanCenter ecosystem for C and C++; the `name` field is a Conan package name.  |
+| `CGA` | Chainguard package ecosystem; the `name` is the name of the package name. The package name might have a `:<RELEASE>` suffix to scope the package to a particular version stream. `<RELEASE>` is a numeric version. |
```

Comment:
This string needs to match exactly what comes in the `package` entry. i.e. "Chainguard" here per

```
"package": {
        "ecosystem": "Chainguard",
        "name": "eks-distro-coredns-1.8",
        "purl": "pkg:apk/chainguard/eks-distro-coredns-1.8"
      },
```

Also, does Chainguard/Wolfi have a concept of source vs binary package names? If so, we should call out which this is here.

Reply:
i will confirm, but i think all is a binary package

Reply:
Thanks. I think we still need to update the first column to be "Chainguard" instead? 

This table is referring to ecosystems, not ID prefixes.

Reply:
ah i see 🤦

Reply:
done